include/zephyr/net/net_ip.h

```diff
@@ -841,6 +841,19 @@ static inline bool net_ipv6_is_ll_addr(const struct in6_addr *addr)
 	return UNALIGNED_GET(&addr->s6_addr16[0]) == htons(0xFE80);
 }
 
+/**
+ * @brief Check if the given IPv6 address is a site local address.
```

Comment:
Commit message talks about "link local" so it should be changed to "site local"?

Reply:
Updated